3-JSS/webpack.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+    mode: 'development',
```

Comment:
usually you would go with a production mode by default for bundling and development settings when working with the dev-server. can you tell the difference?